### ISSUE TITLE
Display # of selected items in status and skill filters on the org page

### ISSUE THREAD

**Issue by ecurrencyhodler** [stakwork/sphinx-tribes#1314]
### Context
After an item has been selected in either the status or skill filter, display the number next to the button.

<img width="167" alt="Screenshot 2024-01-09 at 2 57 05 PM" src="https://github.com/stakwork/sphinx-tribes/assets/32662508/dd31765e-35f6-4fa8-b469-9aa02ab84ccf">

Here is the design in [figma.](https://www.figma.com/file/1aL3lgG5Rpkti4b0j42fwf/Bounties---FINAL-DESIGNS?type=design&node-id=8776%3A5635&mode=design&t=77o2sICWFN65B2Uv-1)

### Acceptance Criteria
- [ ] I've tested on Chrome
- [ ] I've submitted a video of the design
- [ ] I've created a component test that asserts the same # is displayed next to the button as the # of items selected in the status filter
- [ ] I've created a component test that asserts the same # is displayed next to the button as the # of items selected in the skills filter

Here is an [example component test](https://github.com/stakwork/sphinx-tribes/blob/9310f49b3b17a51992dada932f4298eb9eba15ff/frontend/app/src/people/widgetViews/__tests__/AboutView.spec.tsx)

**Comment by ecurrencyhodler** [comment]
Blocked by #1313 and #1315

**Comment by AbdulWahab3181** [comment]
@ecurrencyhodler Could you please assign me this task once it is unblocked?

**Comment by ecurrencyhodler** [comment]
Closing in favor of https://github.com/stakwork/sphinx-tribes-frontend/issues/42
